src/components/IncidentDetails/index.js

```diff
@@ -0,0 +1,175 @@
+import React, { useMemo } from "react";
+import { useForm } from "react-hook-form";
+import dayjs from "dayjs";
+import { IncidentDetailsRow } from "./IncidentDetailsRow";
+import { Select } from "../Select";
+import { IncidentPriority } from "../../constants/incident-priority";
+import {
+  IncidentCommandersOption,
+  IncidentCommandersSingleValue,
+  IncidentPriorityOption,
+  IncidentPrioritySingleValue,
+  IncidentRespondentsMultiValueLabel,
+  IncidentRespondentsOption,
+  priorities
+} from "./select-components";
+import { personRespondents } from "./data";
+
+export const IncidentDetails = ({
+  incident,
+  node,
+  updateStatusHandler,
+  textButton
+}) => {
+  const commandersArray = useMemo(
+    () => [
+      {
+        label: incident.commander_id.name,
+        value: incident.commander_id.id,
+        avatar: incident.commander_id.avatar
+      }
+    ],
+    [incident]
+  );
+
+  const { control, getValues, watch } = useForm({
+    defaultValues: {
+      tracking: "123456",
+      started: dayjs(incident.created_at).format("DD.MM.YYYY") || "01.01.2022",
```

Comment:
```suggestion
      started: incident.created_at,
```

Слабо представляю себе сценарий, когда у полученного с апи инцидента не будет указано created_at значение. даже если там будет undefined - dayjs вернет текущее время и ты его отформируешь. а значит

1. твой фолбек на 01.01.2022 не имеет смысла
2. странный фолбек сам по себе. лучше уж "прочерк" указывать. но не в логике формы. в логике формы должны быть адекватные значения, а форматирование происходит уже во времея рендера в каждом конкретном случае